databroker/src/grpc/kuksa_val_v2/val.rs

```diff
@@ -53,69 +64,297 @@ impl proto::val_server::Val for broker::DataBroker {
                 + 'static,
         >,
     >;
-    
+
     async fn subscribe(
         &self,
-        request: tonic::Request<proto::SubscribeRequest>,
-    ) -> Result<tonic::Response<Self::SubscribeStream>, tonic::Status>
-    {
+        _request: tonic::Request<proto::SubscribeRequest>,
```

Comment:
why include these? is databroker nopt starting if we do not have everything defined?